src/boot/efi/util.c

```diff
@@ -97,6 +97,13 @@ EFI_STATUS efivar_set_int(const EFI_GUID *vendor, CHAR16 *name, UINTN i, BOOLEAN
         return efivar_set(vendor, name, str, persistent);
 }
 
+EFI_STATUS efivar_set_uint64(const EFI_GUID *vendor, CHAR16 *name, UINT64 i, BOOLEAN persistent) {
```

Comment:
See new commits, I moved things around a bit so that we never introduce the string version in the first place and immediately do the binary one. Let me know if switching to binary is actually possible or if we should stick to string for everything. I think I'm missing a bit of context on why strings were used in the first place so switching to binary might be completely the wrong thing to do.